3-extra/1-guess-the-number-project/script.js

```diff
@@ -5,14 +5,25 @@ function guessNumber() {
   let guess = document.querySelector(".inputs-Values").value;
 
   //If the user inputs a bad input ie 0, empty string, number greater that 100, number less than zero Print "Please enter a number between 1 and 100"
-
+if (guess<1||guess>100 || guess==="") {
```

Comment:
you're potentially mixing types here; do you know if `document.querySelector().value` will give you back a number or a string that represents a number (e.g. `"1" instead of `1`). It's worth explicitly casting this to a number to make sure